### PR TITLE
Adds options to customize notifications

### PR DESCRIPTION
**Pull request by mormod**
This PR introduces the possibility to reword the reminder and `:HydrateNow` acknowledgement.

### CODE REVIEW

File: lua/hydrate/init.lua
Comment (stefanlogue) [inline lua/hydrate/init.lua:1]: If we are going to have custom notification messages, we need to ensure these are available for all of the notifications, as currently this does not cover the `print_minutes_until` function

**Review by stefanlogue — CHANGES_REQUESTED** [review]

File: lua/hydrate/init.lua
Comment (mormod) [reply lua/hydrate/init.lua:1]: Should be good to go now

File: CHANGELOG.md
Comment (stefanlogue) [inline CHANGELOG.md:1]: If you can put your changes into the `Unreleased` section and remove the section you created, the release action populates this automatically then

**Review by stefanlogue — CHANGES_REQUESTED** [review]
Everything else looks good and I'll merge this in once this change is made, thanks again for the PR and sorry about turnaround time, been extremely busy!

File: CHANGELOG.md
Comment (mormod) [reply CHANGELOG.md:1]: Same here, I will implement your request later this week!

**Review by stefanlogue — APPROVED** [review]

**Comment by stefanlogue** [comment]
This is now available in `v0.5.0`

**Pull request merged**